homeassistant/components/device_tracker/huawei_hilink.py

```diff
@@ -0,0 +1,217 @@
+"""
+Support for Huawei HiLink routers.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/device_tracker.huawei_hilink/
+"""
+import base64
+import hashlib
+import logging
+from collections import namedtuple
+
+import requests
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.device_tracker import (
+    DOMAIN, PLATFORM_SCHEMA, DeviceScanner)
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+
+REQUIREMENTS = ['defusedxml==0.5.0']
+
+_LOGGER = logging.getLogger(__name__)
+
+_DEFAULT_HOST = '192.168.8.1'
```

Comment:
No need to prefix the constants with `_`.

Reply:
Fixed.